### ISSUE TITLE
Generate pronounceable passwords

### ISSUE THREAD

**Issue by ple103** [bitwarden/browser#119]
I want to be able to generate pronounceable passwords similar to what the LastPass password generator (pwdgen) can do (https://lastpass.com/generate). A feature on top of this I would like to see is the pwdgen replacing letters with numbers and symbols if the either (or both) the numbers and symbols checkbox is selected.

For example:
```
Letter      Replaced with
a           @
e           3
l           1
s           5, $
o, O        0
i           !
```
etc

**Comment by kspearrin** [comment]
If we added something pronounceable, it would be done with passphrase words. Related: https://github.com/bitwarden/mobile/issues/14

I like the number/symbols option though as well.

**Comment by ple103** [comment]
Why not both? Passphrases and pronounceable passwords.
Pronounceable passwords make great random usernames too :)

**Comment by trisk** [comment]
There's quite a lot of literature out there on pronounceable passwords. The most commonly used pronounceable password scheme is probably the old NIST standard [FIPS 181](http://csrc.nist.gov/publications/fips/fips181/fips181.pdf) (which was later found to have [weaknesses](http://www.findravi.com/downloads/ANewAttackonRandomPronounceablePassw.pdf)) as used in apg, followed by the algorithm in the classic [pwgen utility](https://github.com/tytso/pwgen). There's also stuff using newer encoding specs like [Koremutake](http://shorl.com/koremutake.php).

Passphrase generation seems to generally be based on [Diceware](https://en.wikipedia.org/wiki/Diceware). A discussion on implementation considerations: https://discussions.agilebits.com/discussion/54598/old-pronounceable-vs-new-diceware-passwords

**Comment by josh-parris** [comment]
Non-localised, but a starting point: https://github.com/rsanders/correct-horse-battery-staple

with the default dictionary, you get about 10^12 password combinations

**Comment by jnv** [comment]
Related: https://github.com/bitwarden/browser/issues/330, https://github.com/bitwarden/core/issues/93
